src/libraries/System.Reflection/tests/CoreCLR/System.Reflection.CoreCLR.Tests.csproj

```diff
@@ -14,5 +14,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\TestExe\System.Reflection.TestExe.csproj" />
+
+    <TrimmerRootDescriptor Include="$(MSBuildThisFileDirectory)..\ILLink.Descriptors.CoreCLR.xml" />
```

Comment:
Can this `ILLink.Descriptors.CoreCLR.xml` file move next to the .csproj that needs it?